src/main/java/org/folio/dew/batch/bulkedit/jobs/processidentifiers/BulkEditUserIdentifiersJobConfig.java

```diff
@@ -49,7 +50,7 @@
   @Bean
   public Job bulkEditProcessUserIdentifiersJob(JobCompletionNotificationListener listener, Step bulkEditUserStep) {
     return jobBuilderFactory
-      .get(ExportType.BULK_EDIT_IDENTIFIERS.toString().concat("_USER"))
+      .get(ExportType.BULK_EDIT_IDENTIFIERS + "-" + USER.getValue())
```

Comment:
the same - https://github.com/folio-org/mod-data-export-worker/pull/176/files#r841413383

Reply:
fixed